### ISSUE TITLE
Collaborator increases accessibility of issues

### ISSUE THREAD

**Issue by avidrucker** [knowledge-for-good/knowledgeforgood#212]
- [ ] Collaborator marks all unlabeled issues that are questions with the questions label
- [ ] Collaborator marks all unlabeled issues that are tasks with the tasks label
- [ ] Collaborator marks all tasks issues without a time estimate with a suggested time estimate label

**Comment by seekanddefine** [comment]
@avidrucker did you have a specific video in mind here?

**Comment by avidrucker** [comment]
> @avidrucker did you have a specific video in mind here?

Yes, thank you for the reminder. It may have been that I demo'd these things in [this video](https://www.youtube.com/watch?v=pFLOIRbSE5E&t=464s) on our YouTube channel, but too much time has passed and I cannot remember. I would prefer to record three 60 second or less videos instead to make super quick tutorials for the above checklist tasks.

**Comment by seekanddefine** [comment]
I did it! WOOOO! All our issues have been labeled! Yaaaaaassss.
